### ISSUE TITLE
Buttons can disappear upon switch to and from another program

### ISSUE THREAD

**Issue by chrisjj** [microsoft/vscode#90730]
1 Place focus on this button:
![image](https://user-images.githubusercontent.com/6157073/74576226-d4ee8b80-4f81-11ea-87b1-53bfe773a2e8.png)
2 Switch to another program and back (or have e.g. a calendar pop-up take focus)

Expected: button still present, with focus
Observed: button, focus, and focus indicator gone:
![image](https://user-images.githubusercontent.com/6157073/74576262-fd768580-4f81-11ea-9dbc-e88a6cc275a6.png)

Workaround for this specific instance: press Tab, Shift+Tab, Tab.

V1.42.1 on Windows 7.

**Comment by isidorn** [comment]
I can reproduce, and I agree this is not ideal.
The problem is that focus gets back to the ActionBar which controls the button, however the button does not get rendered probably because of some timing dom issue.
I am not sure who owns the actionbar / views title, but this happens due to the exact behavior that actionbar gets hidden when not focus and not hovered.
For now forwarding to @sbatten

fyi @sandy081

**Comment by chrisjj** [comment]
> ActionBar

**Comment by chrisjj** [comment]
Whoops, sorry.

**Comment by chrisjj** [comment]
> The problem is that focus gets back to the ActionBar which controls the button

Not reproduced here.

> however the button does not get rendered 

This is more than just a rendering issue. E.g. no amount of pressing TAB will return to the button.

**Comment by isidorn** [comment]
Reopening since this issue is still present.
Pressing tab + shift tab will bring the button back fyi

**Comment by chrisjj** [comment]
> Reopening since this issue is still present.

I clicked "Close and comment" in error... and could find no way to reopen. Where did I miss it?

**Comment by chrisjj** [comment]
> Pressing tab + shift tab will bring the button back fyi

Indeed, and thanks. 

But another tab is needed to reinstate focus. Added to report.

**Comment by sbatten** [comment]
I cannot reproduce this in master :/.

My steps:
1) Focus the outline view collapse button with keyboard.
2) Alt-Tab away from VS Code
3) Alt-Tab back to VS Code

Button remains focused.

**Comment by isidorn** [comment]
I can easily reproduce this on the mac with latest vscode insiders and out of source
Button remeains visible though, but the focus is not rendered.

**Comment by benibenj** [comment]
If this is an important problem to you, you can use the setting `workbench.view.alwaysShowHeaderActions: true`. If you use that setting you will not run into this problem.

I'm closing this issue as there is a workaround and the focus is essentially still there, it's just not rendered. It's unlikely we will be able to fix the rendering problem.
